Clinical-Genomics/cg
# Adds report version to delivery report

This PR adds report version to the delivery date also it fixes the problem having one delivery report in each bundle and it removes the unnecessary customer parameter option 

How to test 1:
0. install on stage (add instructions on how to as needed)
1. us 
1. `cg upload delivery-report -p grownliger`

Expected outcome:
The report version should be found in the end of the small text under "Leveransrapport..."

How to test 2:
0. install on stage (add instructions on how to as needed)
1. us 
1. find a case that is a rerun
1. `cg upload delivery-report -p [case_id]`

Expected outcome:
The report version should say something like ": 2 ersätter : 1"

How to test 3:
0. install on stage (add instructions on how to as needed)
1. us 
1. find a case that is a rerun and that has a case in hk for a previous bundle
1. `cg upload delivery-report [case_id]`

Expected outcome:
The report should end up in the latest bundle version
